instrumentation/dubbo/dubbo-apache-2.7/library/src/main/java/io/opentelemetry/instrumentation/dubbo/apache/v2_7/client/TracingClientFilter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.dubbo.apache.v2_7.client;
+
+import static io.opentelemetry.instrumentation.dubbo.apache.v2_7.client.DubboInjectAdapter.SETTER;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.api.tracer.utils.NetPeerUtils;
+import io.opentelemetry.instrumentation.dubbo.apache.v2_7.common.DubboHelper;
+import org.apache.dubbo.rpc.Filter;
+import org.apache.dubbo.rpc.Invocation;
+import org.apache.dubbo.rpc.Invoker;
+import org.apache.dubbo.rpc.Result;
+import org.apache.dubbo.rpc.RpcContext;
+import org.apache.dubbo.rpc.RpcException;
+import org.apache.dubbo.rpc.RpcInvocation;
+
+public class TracingClientFilter implements Filter {
+
+  private final DubboClientTracer tracer;
+
+  public TracingClientFilter() {
+    this.tracer = new DubboClientTracer();
+  }
+
+  public static TracingClientFilter newFilter() {
+    return new TracingClientFilter();
+  }
+
+  @Override
+  public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
+    if (invocation instanceof RpcInvocation) {
+      String methodName = invocation.getMethodName();
+      Span span = tracer.startSpan(methodName);
```

Comment:
Span name must be the full name

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#span-name

So I guess it is `invoker.getInterface().getName() + '/' + invocation.getMethodName()`

Reply:
I guess we can add a utility in the helper for it for use in server side too